lib/iris/_constraints.py

```diff
@@ -73,6 +73,27 @@ def __init__(self, name=None, cube_func=None, coord_values=None, **kwargs):
                        model_level_number=[10, 12])
                        & Constraint(ensemble_member=2)
 
+        .. note::
+            Whilst ``&`` is supported, the ``|`` that might reasonably be expected
+            is not. This is because each constraint describes a boxlike region, and
+            thus the intersection of these constraints (obtained with ``&``) will
+            also describe a boxlike region. Allowing the union of two constraints
+            (with the ``|`` symbol) would allow the description of a non-boxlike
+            region. These are difficult to describe with cubes and so it would be
+            ambiguous what should be extracted.
+
+            To generate multiple cubes, each constrained to a different range of
+            the same coordinate, use :py:func:`iris.load_cubes` or
+            :py:func:`iris.cube.CubeList.extract_cubes`.
+
+            A constraint covering multiple ranges of the same coordinate may be
+            used to generate a single cube like so::
```

Comment:
This agrees with the existing pattern for including example code in docstrings found here for example: https://github.com/SciTools/iris/blob/06de63873dcd3108cf3a0b848c0c110e8cc3a7f9/lib/iris/cube.py#L682-L692

Reply:
Is this just confirmation, or an action?

Reply:
Confirmation. I was curious about how this fit into our existing patterns so I decided to check and leave the comment for the record.